Source/WebKit/GPUProcess/media/RemoteImageDecoderAVFProxy.cpp

```diff
@@ -104,7 +106,7 @@ void RemoteImageDecoderAVFProxy::setExpectedContentSize(ImageDecoderIdentifier i
     if (!m_imageDecoders.contains(identifier))
         return;
 
-    m_imageDecoders.get(identifier)->setExpectedContentSize(expectedContentSize);
+    RefPtr { m_imageDecoders.get(identifier) }->setExpectedContentSize(expectedContentSize);
```

Comment:
I wonder whether we should try having HashMap::get() return a RefPtr instead of a raw pointer.
Not in this patch though.

Reply:
Oh interesting. I didn't realize that HashMap::get() doesn't return the exact type of the values.